os/mm/umm_heap/umm_sem.c

```diff
@@ -97,7 +97,7 @@
 
 int umm_trysemaphore(void)
 {
-	return mm_trysemaphore(USR_HEAP);
+	return mm_trysemaphore(BASE_HEAP);
```

Comment:
need to discuss.
When we split heap regions, should we block to use heap for all?